### ISSUE TITLE
`cupy` icons for download counts, versions, and platforms not working?

### ISSUE THREAD

**Issue by leofang** [conda-forge/cupy-feedstock#270]
I think this started since CuPy v13.0.0 updates. I was hoping it's a transient glitch, but it seems the icons for `cupy` are still not working today (while those for `cupy-core` do work), see the screenshot below (viewed on macOS Safari 16.6.1):
<img width="921" alt="截圖 2024-04-28 下午9 05 58" src="https://github.com/conda-forge/cupy-feedstock/assets/5534781/90c024c7-b207-409e-92f4-6927d92bf5aa">

The following icons are problematic: 
- https://img.shields.io/conda/dn/conda-forge/cupy.svg (download counts)
- https://img.shields.io/conda/vn/conda-forge/cupy.svg (versions)
- https://img.shields.io/conda/pn/conda-forge/cupy.svg (platforms)

**Comment by leofang** [comment]
cc: @jakirkham for vis (in case you know who to reach out to... doesn't seem like it's within conda-forge core's control?)

**Comment by leofang** [comment]
cc: @kmaehashi for vis (this also affects the icon on CuPy's repo landing page (the same icon is used in `README.md`)

**Comment by kmaehashi** [comment]
It seems the badge uses [the anaconda API](https://github.com/badges/shields/blob/46ed6f02db0c4fe1546922221bf1a9340eab0eaf/services/conda/conda-base.js#L23). Just a wild guess but shileds.io has some parse size/time limit?

```
% curl -s https://api.anaconda.org/package/conda-forge/cupy | wc
  429732  656384 10675675
% curl -s https://api.anaconda.org/package/conda-forge/numpy | wc
  211623  332622 5539668
```

**Comment by jakirkham** [comment]
Yeah can notice the slow loading just using the browser to visit: https://anaconda.org/conda-forge/cupy

Something odd is happening there. Raised issue: https://github.com/conda/infrastructure/issues/928
